portfolio/target/portfolio-1/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Team Creator</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Why is all this stuff inside a portfolio folder? Not sure what is going on with the file structure

Reply:
every time i run the page with appengine the target file automatically updates